cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py

```diff
@@ -181,21 +181,27 @@ def test_failover(cfy, hosts, ha_hello_worlds, logger):
     _test_hellos(ha_hello_worlds)
 
 
-def test_remove_manager_from_cluster(cfy, hosts, ha_hello_worlds, logger):
-    ha_helper.set_active(hosts.instances[1], cfy, logger)
-    ha_helper.delete_active_profile()
+def test_remove_manager_from_cluster(cfy, hosts, ha_hello_worlds, logger,
+                                     ignore_node_host_ip=None,
+                                     delete_profile=True):
+    ha_helper.set_active(hosts.instances[1], cfy, logger,
+                         ignore_node_host_ip=ignore_node_host_ip)
+    if delete_profile:
+        ha_helper.delete_active_profile()
```

Comment:
delete_active_profile calls are likely not needed, retest without them and remove if passes

Reply:
I had to put it there because the test deletes the active profile and then tries to use the joined nodes, which causes the command to fail because it is not the active node in the cluster.
I tried it with deleting the active profile and it failed.